subxt/src/rpc.rs

```diff
@@ -31,15 +31,13 @@ use crate::{
     storage::StorageKeyPrefix,
     Config,
     Metadata,
+    PhantomDataSendSync,
 };
 use codec::{
     Decode,
     Encode,
 };
-use core::{
-    convert::TryInto,
-    marker::PhantomData,
-};
+use core::convert::TryInto;
```

Comment:
Can we get rid of this import too now because edition 2021?

Reply:
Turns out we can :)